optaplanner-docs/src/main/asciidoc/PlannerIntroduction/DownloadAndRunTheExamples/DownloadAndRunTheExamples-section.adoc

```diff
@@ -63,7 +63,7 @@ To run the examples in your favorite IDE:
 
 
 [[useWithMavenGradleEtc]]
-== Use OptaPlanner with Maven, Gradle, Ivy, Buildr, or ANT
+== Use OptaPlanner with Maven, Gradle, or ANT
```

Comment:
Are we keeping ANT then? I didn't notice any `build.xml` in our projects.

Reply:
I was thinking of removing ANT too - nobody uses that any more - but we still need to talk about plain jar files that students drop in their IDE without maven or gradle.